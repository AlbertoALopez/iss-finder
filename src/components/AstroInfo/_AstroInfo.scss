#slider {
  height: 100px;
  font-size: 20px;
  .fact {
    // font-size: 26px;
  }
}
