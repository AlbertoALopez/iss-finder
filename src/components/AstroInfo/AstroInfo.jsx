import React, { Component } from 'react';
import { Segment } from 'semantic-ui-react';
import Slider from 'react-slick';
import './_AstroInfo.scss';


export default class AstroInfo extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    const settings = {
      autoplay: true,
      autoplaySpeed: 4000,
      dots: false,
      infinite: true,
      speed: 500,
      slidesToShow: 1,
      slidesToScroll: 1,
    };
    return (
      <Segment>
        <div id="slider">
          <p>Did you know?</p>
          <Slider {...settings}>
            <span className="fact"><p>The ISS travels at 27,600 kilometres per hour</p></span>
            <span className="fact"><p>The ISS has an orbit height of 400 kilometres</p></span>
            <span className="fact"><p>The ISS makes a full rotation of the earth every one and half hours</p></span>
            <span className="fact"><p>The ISS was launched on November 20, 1988</p></span>
            <span className="fact"><p>The ISS has an orbit height of 400 kilometres</p></span>
          </Slider>
        </div>
      </Segment>
    );
  }
}
