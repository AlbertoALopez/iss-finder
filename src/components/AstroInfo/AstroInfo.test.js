import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AstroInfo from './AstroInfo.jsx';

window.matchMedia = window.matchMedia || function() {
    return {
        matches : false,
        addListener : function() {},
        removeListener: function() {}
    };
};

test('AstroInfo renders properly', () => {
  const component = shallow(
    <AstroInfo />
  );

  expect(toJson(component)).toMatchSnapshot();
});
