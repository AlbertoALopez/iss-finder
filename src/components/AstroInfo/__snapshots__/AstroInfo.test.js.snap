exports[`test AstroInfo renders properly 1`] = `
<Segment>
  <div
    id="slider">
    <p>
      Did you know?
    </p>
    <Slider
      autoplay={true}
      autoplaySpeed={4000}
      dots={false}
      infinite={true}
      slidesToScroll={1}
      slidesToShow={1}
      speed={500}>
      <span
        className="fact">
        <p>
          The ISS travels at 27,600 kilometres per hour
        </p>
      </span>
      <span
        className="fact">
        <p>
          The ISS has an orbit height of 400 kilometres
        </p>
      </span>
      <span
        className="fact">
        <p>
          The ISS makes a full rotation of the earth every one and half hours
        </p>
      </span>
      <span
        className="fact">
        <p>
          The ISS was launched on November 20, 1988
        </p>
      </span>
      <span
        className="fact">
        <p>
          The ISS has an orbit height of 400 kilometres
        </p>
      </span>
    </Slider>
  </div>
</Segment>
`;
